modules/ROOT/pages/mule-error-concept.adoc

```diff
@@ -80,7 +80,19 @@ HTTP GET on resource 'http://jsonplaceholder.typicode.com:80/mybadrequest' faile
 ----
 +
 * `#[error.detailedDescription]` returns `HTTP GET on resource 'http://jsonplaceholder.typicode.com:80/mybadrequest' failed: not found (404).`
-* `#[error.errorType]` returns `HTTP:NOT_FOUND`
+* `#[error.errorType]` returns 
```

Comment:
Please note that all the other selector examples are showing log outputs (and not the actual underlying structure of the selector result). The log output of #[error.errorType] is "HTTP:"NOT_FOUND".